tests/tensor/test_math.py

```diff
@@ -2187,8 +2189,9 @@ def test_broadcastable1(self):
         rng = np.random.default_rng(seed=utt.fetch_seed())
         x = TensorType(dtype=config.floatX, shape=(1, None, None))("x")
         y = tensor3("y")
-        z = tensordot(x, y)
+        z = tensordot(x, y, axes=2)
```

Comment:
This is the default right? Shouldn't need to change the test

Reply:
Shoot, I changed the default with respect to numpy. I thought that `numpy` used `axes=1` as default. I'll change it back to 2.